src/crypto/crypto_context.h

```diff
@@ -142,6 +147,7 @@ class SecureContext final : public BaseObject {
   SSLCtxPointer ctx_;
   X509Pointer cert_;
   X509Pointer issuer_;
+  X509_STORE* owned_cert_store_cached_ = nullptr;
```

Comment:
non-blocking nit: A short comment here about who owns this pointer and who is responsible for cleaning it up would be good for others coming into the code later.

Reply:
Sure, done in 509156ac75aed153b3deab72b33eec6c3e6efe17 👍